media-libs/oidn/oidn-1.4.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{6,7,8,9} )
+
+inherit cmake python-single-r1 eutils
+
+DESCRIPTION="Intel(R) Open Image Denoise library"
+HOMEPAGE="http://www.openimagedenoise.org/"
+
+if [[ ${PV} = *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/OpenImageDenoise/oidn.git"
+	EGIT_BRANCH="master"
+else
+	SRC_URI="https://github.com/OpenImageDenoise/${PN}/releases/download/v${PV}/${P}.src.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	dev-cpp/tbb
+	dev-lang/ispc"
+DEPEND="
+	${RDEPEND}
+	dev-util/cmake"
+
+CMAKE_BUILD_TYPE=Release
```

Comment:
We usually prefer not to have to set this (instead patch the CMakeLists.txt file(s) to be tolerant of other release types) because the Gentoo build type sets a bunch of important variables.

Reply:
I think I looked into this in the past and gave up because probably at that point it was a PITA to patch this out.
I don't remember the specifics though :C

Reply:
Look into it next time? 😁